src/app/core/services/github.service.ts

```diff
@@ -30,7 +30,7 @@ import { ElectronService } from './electron.service';
 import { ERRORCODE_NOT_FOUND, ErrorHandlingService } from './error-handling.service';
 import { LoggingService } from './logging.service';
 
-const Octokit = require('@octokit/rest');
+const { Octokit } = require('@octokit/rest');
```

Comment:
Require call may be converted to an import

import { Octokit } from '@octokit/rest';